integration-tests/vertx-graphql/src/test/java/io/quarkus/vertx/graphql/it/VertxGraphqlTest.java

```diff
@@ -37,12 +31,14 @@ public void testGraphQlQuery() {
 
     @Test
     public void testWebSocketSubProtocol() throws Exception {
+        Vertx vertx = Vertx.vertx();
         HttpClient httpClient = vertx.createHttpClient();
         WebSocketConnectOptions options = new WebSocketConnectOptions().setPort(getPortFromConfig())
                 .addSubProtocol("graphql-ws").setURI("/graphql");
         CompletableFuture<Boolean> wsFuture = new CompletableFuture<>();
         httpClient.webSocket(options, event -> wsFuture.complete(event.succeeded()));
         Assertions.assertTrue(wsFuture.get(1, TimeUnit.MINUTES));
+        vertx.close();
```

Comment:
No, that won't be enough. 
Closing a vert.x instance is asynchronous, so you need a countdown latch and use the async version.
In addition, if the test fails, you won't close the vert.x instances and resources will still be allocated (event loops...)

Reply:
Ouch, this slipped in fast. I'll update this part. Thanks